assets/javascripts/modules/validatorFocus.js

```diff
@@ -7,10 +7,26 @@
     }
 
     $('.error-summary a').on('click', function(e) {
+      var $this = $(this);
+      var focusId = $this.attr('data-focuses');
+      if (!focusId)
+      {
+        focusId = ("" + $this.attr("href")).trim();
```

Comment:
I suppose the thing that feels weird here is your using a href `focusId = $this.attr("href");` to then select an element via `var inputToFocus = $(focusId);` that swapping around of use cases may be the thing causing issues here.